app/src/main/java/at/roteskreuz/stopcorona/model/repositories/ReportingRepository.kt

```diff
@@ -189,12 +187,13 @@ class ReportingRepositoryImpl(
                 infectionMessengerRepository.storeSentInfectionMessages(resetMessages)
             }
 
-            infectionMessages.addAll(nearbyRecordDao.observeRecordsRecentThan(thresholdTime)
-                .blockingFirst()
-                .map { nearbyRecord ->
-                    nearbyRecord.publicKey to InfectionMessageContent(infectionLevel, nearbyRecord.timestamp)
-                }
-            )
+            // TODO: 28/05/2020 dusanjencik: How to integrate exposure notifications to infection messages?
```

Comment:
https://wiki.sinnerschrader.com/display/SWP/Best+practices#UNUSED

Reply:
If there is todo comment, some code might be commented. Will be fixed in later PR.